toolchain/parse/tree.h

```diff
@@ -106,6 +106,8 @@ class Tree : public Printable<Tree> {
   // Tests whether there are any errors in the parse tree.
```

Comment:
While we're here I think it would be helpful to clarify what exactly constitutes an error in this context, e.g.

```
// Returns true if this tree represents a syntactically invalid input.
```

(Sorry, GH won't let me suggest an edit here)

Reply:
I've removed this comment, and worked on the comment on has_errors_ (I think we typically don't have comments on accessors like this, and I'd rather only have one comment to update). Trying to also reflect better my comments above about the flag's semantic.